pkg/config/app_config.go

```diff
@@ -214,6 +214,7 @@ func GetDefaultConfig() []byte {
 		`gui:
   ## stuff relating to the UI
   scrollHeight: 2
+  scrollPastBottom: true
```

Comment:
Hmm since lazygit allowed scroll past the bottom - means it's not bug, it's intended -, i thought the behaviour of this should be same before/after this commit for UI.

Reply:
Very good point, consider my comment discarded